tests/aws/services/cloudformation/api/test_stacks.py

```diff
@@ -340,6 +340,7 @@ def test_update_stack_with_same_template_withoutchange_transformation(
         )
 
     @markers.aws.validated
+    @skip_if_legacy_engine()
```

Comment:
🎉 it's great to see more skips for the old engine meaning that the new engine is a higher parity!